src/math/src/tests/test_keccak256.cairo

```diff
@@ -0,0 +1,75 @@
+use alexandria_math::keccak256::keccak256;
+use debug::PrintTrait;
+
+#[test]
+#[available_gas(2000000)]
+fn test_keccak256_empty_bytes() {
+    let input = array![];
+
+    let hash = keccak256(input);
+
+    assert(
+        hash == 0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470,
+        'wrong hash value'
+    )
+}
```

Comment:
can you really hash nothing ?

Reply:
yes, see https://emn178.github.io/online-tools/keccak_256.html and https://ethereum.stackexchange.com/questions/35103/keccak-hash-of-null-values-result-in-different-hashes-for-different-types